docs/userguides/platform.md

```diff
@@ -0,0 +1,179 @@
+If you haven't yet, please sign up for Silverback at [https://silverback.apeworx.io](https://silverback.apeworx.io).
+
+Once you have signed up, you can actually create (and pay for) your Clusters from the Silverback CLI utility by first
+logging in to the Platform using [`silverback login`][silverback-login],
+and then using [`silverback cluster new`][silverback-cluster-new] to follow the steps necessary to deploy it.
+
+```{note}
+The Platform UI will let you create and manage Clusters using a graphical experience, which may be preferred.
+The CLI experience is for those working locally who don't want to visit the website, or are locally developing their applications.
+```
+
+## Connecting to your Cluster
+
+To connect to a cluster, you can use commands from the [`silverback cluster`][silverback-cluster] subcommand group.
+For instance, to list all your available bots on your cluster, use [`silverback cluster bots list`][silverback-cluster-bots-list].
+To obtain general information about your cluster, just use [`silverback cluster info`][silverback-cluster-info],
+or [`silverback cluster health`][silverback-cluster-health] to see the current status of your Cluster.
+
+If you have no bots, we will first have to containerize our Applications and upload them to a container registry that our Cluster is configured to access.
+
+```{note}
+Building a container for your application can be an advanced topic, we have included the `silverback build` subcommand to help assist in generating Dockerfiles.
+```
+
+## Building your Bot
+
+TODO: Add build process and describe `silverback build --autogen` and `silverback build --upgrade`
+
+TODO: Add how to debug containers using `silverback run` w/ `taskiq-redis` broker
+
+## Adding Environment Variables
+
+Once you have created your bot application container image, you might know of some environment variables the image requires to run properly.
+Thanks to it's flexible plugin system, ape plugins may also require specific environment variables to load as well.
+Silverback Clusters include an environment variable management system for exactly this purpose,
+which you can manage using [`silverback cluster vars`][silverback-cluster-vars] subcommand.
+
+The environment variable management system makes use of a concept called "Variable Groups" which are distinct collections of environment variables meant to be used together.
+These variable groups will help in managing the runtime environment of your Applications by allowing you to segregate different variables depending on each bot's needs.
+
+To create an environment group, use the [`silverback cluster vars new`][silverback-cluster-vars-new] command and give it a name and a set of related variables.
+For instance, it may make sense to make a group of variables for your favorite Ape plugins or services, such as RPC Providers, Blockchain Data Indexers, Etherscan, etc.
+You might have a database connection that you want all your bots to access.
```

Comment:
It would be nice to have the ability to use .yaml for this. You may have a lot of vars you'll want to create, adding those with a CLI could be painful and prone to error

Reply:
I wanted to make some way to do env var reading, essentially `-e ENV_VAR_NAME` without `=...` will load whatever is in your environment already

Reply:
fwiw, this mirrors the docker CLI. Though a cool feature in the future might be allowing users to specify an env file.

Reply:
> fwiw, the `-e VAR=val` syntax mirrors the docker CLI. Though a cool feature in the future might be allowing users to specify an env file.

That would also work fine too, extra feature that looks the same to the client after processing it (list of env vars)

I'm gonna try and do the `-e VAR` now w/o the `=` as that seems very useful and reduces annoyance at having to type `-e VAR="${VAR}"` constantly

Reply:
Pulling from the environment is unexpected behavior and could be a security issue. I'd recommend against that.

Reply:
You've opted-in to it though by writing it? `-e VAR` is a shortcut for `-e VAR="${VAR}"`

Reply:
We should set up encoding/decoding for secrets, but that's for another day